backend/app/api/endpoints/admin/public_bots.py

```
@@ -307,6 +312,7 @@ def _create_public_ghost(
     system_prompt: str,
     mcp_servers: dict,
     skills: list,
+    default_knowledge_base_refs: list = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)